modules/ppcp-wc-gateway/src/Settings/Fields/connection-tab-fields.php

```diff
@@ -496,6 +497,42 @@
 			'requirements' => array(),
 			'gateway'      => Settings::CONNECTION_TAB_ID,
 		),
+		'subtotal_mismatch_behavior'                    => array(
+			'title'        => __( 'Subtotal mismatch behavior', 'woocommerce-paypal-payments' ),
+			'type'         => 'select',
+			'input_class'  => array( 'wc-enhanced-select' ),
+			'default'      => 'vertical',
+			'desc_tip'     => true,
+			'description'  => __(
+				'Differences between WooCommerce and PayPal roundings may give origin to a mismatch in order items subtotal calculations. If not handled these mismatches will cause the PayPal transaction to fail.',
```

Comment:
Maybe should add comma after `not handled`.

And maybe just `... may cause mismatch in ...`?

Reply:
Ok will do.